internal/auth-service/auth-service-config.go

```diff
@@ -0,0 +1,34 @@
+package auth_service
+
+import (
+	configv1alpha1 "github.com/liqotech/liqo/apis/config/v1alpha1"
+	"github.com/liqotech/liqo/pkg/clusterConfig"
+)
+
+func (authService *AuthServiceCtrl) GetAuthServiceConfig(kubeconfigPath string) error {
+	waitFirst := make(chan bool)
+	isFirst := true
+	go clusterConfig.WatchConfiguration(func(configuration *configv1alpha1.ClusterConfig) {
+		authService.handleConfiguration(configuration.Spec.AuthConfig)
+		if isFirst {
+			waitFirst <- true
+			isFirst = false
+		}
+	}, nil, kubeconfigPath)
+	<-waitFirst
+	close(waitFirst)
+
+	return nil
+}
+
+func (authService *AuthServiceCtrl) handleConfiguration(config configv1alpha1.AuthConfig) {
+	authService.configMutex.Lock()
+	defer authService.configMutex.Unlock()
+	authService.config = config
```

Comment:
I think you should use `deepcopy`, because of the pointers in the AuthConfig struct.

Reply:
I don't think so, because AuthConfig is a struct passed by value with only a boolean as filed at the moment

But I'm adding the DeepCopy for possible future additions in this struct

Reply:
You are right, I did not check the struct :grimacing: . Anyway, if we plan to add more fields, a `deepcopy` would be better (I already faced the lacking of `deepcopy` bug and it was very hard to troubleshoot)